core/controllers/mappings.py

```diff
@@ -101,9 +100,9 @@ def lookup_fk_row(model: db.Model, lookup_field: str, lookup_val: Any) -> str |
             "Programme ID": "programme_id",
             "Programme Name": "programme_name",
             "FundType_ID": "fund_type_id",
-            "Organisation ID": "organisation_id",
+            "Organisation": "organisation",
         },
-        fk_relations=[("organisation_id", ents.Organisation, "organisation_id", "organisation_id")],
+        fk_relations=[("organisation_name", ents.Organisation, "organisation_id", "organisation")],
```

Comment:
should item 3 in this tuple now be "organisation_name"?

Reply:
no, because `Programme.organisation_id` is what we call the FK attribute that relates to the `Organisation.id`